index.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  Bridge: require('./lib/bridge.js'),
-  Peer: require('./lib/peer.js'),
-  PeerGroup: require('./lib/peerGroup.js')
+  Bridge: require('./src/bridge.js'),
+  Peer: require('./src/peer.js'),
+  PeerGroup: require('./src/peerGroup.js')
```

Comment:
Sorry, this one shouldn't be here. It's just because I'm using my fork and it's not working with /lib/ in there.